src/battle_script_commands.c

```diff
@@ -1276,8 +1276,8 @@ static void atk04_critcalc(void)
                 + 2 * (holdEffect == HOLD_EFFECT_LUCKY_PUNCH && gBattleMons[gBattlerAttacker].species == SPECIES_CHANSEY)
                 + 2 * (holdEffect == HOLD_EFFECT_STICK && gBattleMons[gBattlerAttacker].species == SPECIES_FARFETCHD);
 
-    if (critChance > 4)
-        critChance = 4;
+    if (critChance > ARRAY_COUNT(sCriticalHitChance) - 1 )
+    critChance = ARRAY_COUNT(sCriticalHitChance) - 1;
```

Comment:
Please fix this indentation error.  Everything else looks fine.

Reply:
Nice catch. Fixed!

Reply:
You could also change if (critChance >= ARRAY_COUNT(sCriticalHitChance)), makes more sense this way imo

Reply:
@DizzyEggg That's what I thought too, but I had figured it wouldn't match. Turns out it does! I didn't realize the compiler worked that way. Thanks!